CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 **Note**: Numbers like (\#123) point to closed Pull Requests on the fractal-server repository.
 
-
 # 1.4.3 (unreleased)
 
+> **WARNINGs**:
+>
+> 1. This update requires running a fix-db script, available at https://raw.githubusercontent.com/fractal-analytics-platform/fractal-server/main/scripts/fix_db/current.py.
```

Comment:
This URL is risky, cause it does not include the version tag. It's best to use a versioned URL, as we did for 1.4.2.

Reply:
Not having a 1.4.3 release yet, I had thought of leaving `main` until the release is ready

Reply:
You're right, let's wait.

Also, for the record, as of https://github.com/fractal-analytics-platform/fractal-server/issues/1094#issuecomment-1893931516 we will likely move away from `current.py` and write the version name in the Python module itself - so that this risk (downloading or running the wrong script) won't be there any more.